--- Makefile.am.old	2008-11-10 03:44:56.000000000 +0100
+++ Makefile.am	2008-11-10 03:45:24.000000000 +0100
@@ -6,6 +6,7 @@
     $(GTK_CFLAGS)           \
     $(GTKSOURCEVIEW_CFLAGS) \
     $(WEBKIT_CFLAGS)        \
+    $(LIBXML_CFLAGS)        \
     -I../katze
 
 AM_CFLAGS = -DMIDORI_LOCALEDIR=\""$(localedir)"\"
@@ -17,6 +18,7 @@
     $(GTKSOURCEVIEW_LIBS)\
     $(WEBKIT_LIBS)       \
     $(INTLLIBS)          \
+    $(LIBXMLÃ_LIBSÃ)       \
     ../katze/libkatze.la
 
 bin_PROGRAMS = \
