--- ./etc/syslog.conf.orig	Tue Mar 17 21:25:31 2009
+++ ./etc/syslog.conf	Tue Mar 17 21:19:44 2009
@@ -7,3 +7,4 @@
 ROTATEGENS=1			# file: keep X generations of rotated logs (busybox 1.2+)
 REMOTE=loghost:514		# remote: where to log
 FOREGROUND=no			# run in foreground (don't use!)
+KLOGD_LOGLEVEL=1		# the default log level of console messages
--- ./etc/init.d/syslog.orig	Tue Mar 17 21:22:37 2009
+++ ./etc/init.d/syslog	Tue Mar 17 21:23:24 2009
@@ -6,6 +6,7 @@
 set -e
 
 if [ -f /etc/syslog.conf ]; then
+	KLOGD_LOGLEVEL=""
 	. /etc/syslog.conf
 	LOG_LOCAL=0
 	LOG_REMOTE=0
@@ -38,6 +39,9 @@
 	if [ "$REDUCE" = "yes" ]; then
 		SYSLOG_ARGS="$SYSLOG_ARGS -S"
 	fi
+	if [ -n "$KLOGD_LOGLEVEL" ]; then
+		KLOGD_ARGS="$KLOGD_ARGS -c $KLOGD_LOGLEVEL"
+	fi
 else
 	# default: log to 16K shm circular buffer
 	SYSLOG_ARGS="-C"
@@ -47,7 +51,7 @@
   start)
 	echo -n "Starting syslogd/klogd: "
 	start-stop-daemon -S -b -n syslogd -a /sbin/syslogd -- -n $SYSLOG_ARGS
-	start-stop-daemon -S -b -n klogd -a /sbin/klogd -- -n
+	start-stop-daemon -S -b -n klogd -a /sbin/klogd -- -n $KLOGD_ARGS
 	echo "done"
 	;;
   stop)
