diff --git a/recipes/python/python-elementary_svn.bb b/recipes/python/python-elementary_svn.bb
index 08def2a..9315a60 100644
--- a/recipes/python/python-elementary_svn.bb
+++ b/recipes/python/python-elementary_svn.bb
@@ -4,8 +4,6 @@ RDEPENDS += "python-evas python-ecore python-edje"
 PV = "0.1+svnr${SRCREV}"
 PR = "ml1"
 
-SRC_URI += "file://py_elm.patch;patch=1"
-
 do_stage() {         
 	distutils_stage_all 
 }  
