From 42e05501fffdf67e47d62df5335352d32c5dccbe Mon Sep 17 00:00:00 2001
From: Klaus Kurzmann <mok@fluxnetz.de>
Date: Sat, 13 Dec 2008 10:49:31 +0100
Subject: [PATCH] dbus: start at 3

Signed-off-by: Klaus Kurzmann <mok@fluxnetz.de>
---
 packages/dbus/dbus.inc |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/packages/dbus/dbus.inc b/packages/dbus/dbus.inc
index 64ef21c..faca9b1 100644
--- a/packages/dbus/dbus.inc
+++ b/packages/dbus/dbus.inc
@@ -88,4 +88,4 @@ chmod u+s /usr/libexec/dbus-daemon-launch-helper
 CONFFILES_${PN} = "${sysconfdir}/dbus-1/system.conf ${sysconfdir}/dbus-1/session.conf"
 
 INITSCRIPT_NAME = "dbus-1"
-INITSCRIPT_PARAMS = "defaults"
+INITSCRIPT_PARAMS = "defaults 3"
-- 
1.6.0.6

