From dfba2db6f31d27ebed6d700bff5f20f81eab42fb Mon Sep 17 00:00:00 2001
From: Klaus Kurzmann <mok@fluxnetz.de>
Date: Sat, 13 Dec 2008 10:40:48 +0100
Subject: [PATCH] opkg: Do not run opkg-cl configure everytime.

Author: Chia-I Wu <olv@openmoko.com>
Signed-off-by: Klaus Kurzmann <mok@fluxnetz.de>
---
 packages/opkg/opkg_svn.bb |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/packages/opkg/opkg_svn.bb b/packages/opkg/opkg_svn.bb
index 24547d6..59cb39e 100644
--- a/packages/opkg/opkg_svn.bb
+++ b/packages/opkg/opkg_svn.bb
@@ -16,7 +16,9 @@ if [ "x$D" != "x" ]; then
 	install -d ${IMAGE_ROOTFS}/${sysconfdir}/rcS.d
 	# this happens at S98 where our good 'ole packages script used to run
 	echo "#!/bin/sh
-opkg-cl configure
+if ! test -f /etc/.configured; then
+	opkg-cl configure
+fi
 rm -f /${sysconfdir}/rcS.d/S${OPKG_INIT_POSITION}configure
 " > ${IMAGE_ROOTFS}/${sysconfdir}/rcS.d/S${OPKG_INIT_POSITION}configure
 	chmod 0755 ${IMAGE_ROOTFS}${sysconfdir}/rcS.d/S${OPKG_INIT_POSITION}configure
-- 
1.6.0.4

