From 015bebd52b4144fa9a05e7cedeb420b468c6357e Mon Sep 17 00:00:00 2001
From: Klaus Kurzmann <mok@fluxnetz.de>
Date: Mon, 8 Dec 2008 17:08:38 +0100
Subject: [PATCH] navit: add --enable-svg2png-scaling-flag=32 to EXTRA_OECONF

---
 recipes/navit/navit.inc |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/recipes/navit/navit.inc b/recipes/navit/navit.inc
index 8db0595..e76aba7 100644
--- a/recipes/navit/navit.inc
+++ b/recipes/navit/navit.inc
@@ -6,7 +6,7 @@ RRECOMMENDS = "gpsd speechd flite"
 
 inherit autotools
 
-EXTRA_OECONF = "--disable-binding-python --disable-gui-sdl --disable-samplemap --enable-avoid-float --enable-avoid-unaligned"
+EXTRA_OECONF = "--disable-binding-python --disable-gui-sdl --disable-samplemap --enable-avoid-float --enable-avoid-unaligned  --enable-svg2png-scaling-flag=32"
 
 PACKAGES = "${PN}-dbg ${PN}-dev ${PN} ${PN}-doc ${PN}-locale"
 
-- 
1.6.0.4

