From 11839f02ab0f852d3cffd0be0a9098b98258a597 Mon Sep 17 00:00:00 2001
From: Klaus Kurzmann <mok@fluxnetz.de>
Date: Mon, 8 Dec 2008 17:09:27 +0100
Subject: [PATCH] speech-dispatcher: configure without pulseaudio

---
 .../speech-dispatcher/speech-dispatcher_0.6.6.bb   |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/recipes/speech-dispatcher/speech-dispatcher_0.6.6.bb b/recipes/speech-dispatcher/speech-dispatcher_0.6.6.bb
index 105c323..145f075 100644
--- a/recipes/speech-dispatcher/speech-dispatcher_0.6.6.bb
+++ b/recipes/speech-dispatcher/speech-dispatcher_0.6.6.bb
@@ -16,7 +16,7 @@ SRC_URI = "http://www.freebsoft.org/pub/projects/speechd/${PN}-${PV}.tar.gz \
 	   file://configSpeechd.conf.in_00.patch;patch=1"
 
 LEAD_SONAME = "libspeechd.so"
-EXTRA_OECONF = " --with-espeak --with-flite --without-ibmtts --without-nas --with-alsa --with-pulse "
+EXTRA_OECONF = " --with-espeak --with-flite --without-ibmtts --without-nas --with-alsa "
 
 INITSCRIPT_NAME = "speech-dispatcher"
 INITSCRIPT_PARAMS = "defaults 45"
-- 
1.6.0.4

